gallery/src/DarkModeModal/DarkModeModal.tsx

```diff
@@ -0,0 +1,111 @@
+ */
+import React, { useEffect, useState } from 'react';
+
+import {
+  NxButton,
+  NxModal,
+  NxForm,
+  NxFormGroup,
+  NxToggle,
+  useToggle,
+  NxFieldset,
+  NxRadio,
+  NxTooltip,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+
+const DarkModeModal = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [enableModeChange, toggleEnableModeChange] = useToggle(false);
+  const [mode, setMode] = useState<string | null>(null);
+
+  const { classList } = document.documentElement;
+
+  useEffect(() => {
+    classList.toggle('nx-html--enable-color-schemes', enableModeChange);
+  }, [enableModeChange]);
+
+  useEffect(() => {
+    classList.toggle('nx-html--dark-mode', mode === 'dark' && enableModeChange);
+    classList.toggle('nx-html--light-mode', mode === 'light' && enableModeChange);
+  }, [enableModeChange, mode]);
+
+  return (
+    <>
+      <NxButton variant="tertiary" onClick={() => setShowModal(true)}>Theme Settings</NxButton>
+      { showModal &&
+        <NxModal id="nx-modal-dark-mode-example"
+                 onCancel={modalCloseHandler}
+                 aria-labelledby="nx-modal-dark-mode-form-header">
+          <NxForm onSubmit={modalCloseHandler}
+                  submitBtnText= "Save and Close">
+            <header className="nx-modal-header">
+              <h2 className="nx-h2" id="nx-modal-dark-mode-form-header">
+                Setting Preferences for Light or Dark Mode
+              </h2>
+            </header>
+            <div className="nx-modal-content">
+              <NxFormGroup label="Enable Theme Changes">
+                <NxToggle onChange={toggleEnableModeChange} isChecked={enableModeChange}>
+                  Opt-in to Allow Theming
+                  <NxTooltip title="Opting in will allow you to make adjustments to the display theme. Failure
+                             to do so will always result in light mode. This can be done by adding the
+                             'nx-html--enable-color-schemes' class to the HTML element.">
+                    <NxFontAwesomeIcon className="dark-mode-modal__info-icon" icon={faInfoCircle}/>
+                  </NxTooltip>
+                </NxToggle>
+              </NxFormGroup>
+              <NxFieldset label="Choose Your Mode">
+                <NxRadio name="mode"
+                         value="browserChoice"
+                         onChange={setMode}
+                         isChecked={mode === 'browserChoice'}
+                         disabled = {!enableModeChange}>
+                  Let Your Browser Color Preference Decide
+                  <NxTooltip title="If theme changes are permitted, the default theme will be dictated by
+                             your browser or OS theme choice. The prefers-color-scheme media query registers
+                             this choice.">
```

Comment:
I think we can remove the conditional aspect of this sentence.  The other two radios aren't described conditionally, and IMO the fact that it's conditional is made clear by the radios becoming disabled when the Opt-In toggle is off.

Reply:
> I think we can remove the conditional aspect of this sentence. The other two radios aren't described conditionally, and IMO the fact that it's conditional is made clear by the radios becoming disabled when the Opt-In toggle is off.

Absolutely, changed here: [3f5e143](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/3f5e1437e88f0edbdce9ae5bd7708d4676a6d0f4)